rosidl_generator_py/resource/_msg.py.em

```diff
@@ -212,6 +212,19 @@ class @(spec.base_type.type)(metaclass=Metaclass):
         'int64', 'uint64',
         'string',
     ]]@
+@[    if field.type.type.startswith('int')]@
+@{
+nbytes = int(field.type.type[field.type.type.rfind('t') + 1:])
```

Comment:
`nbytes` should be `nbits`.

Instead of `field.type.type.rfind('t') + 1` you can just use `4` since you already ensured the `startswith` before. (`5` in the `uint` case below.)